ingestion/src/metadata/ingestion/lineage/sql_lineage.py

```diff
@@ -163,6 +171,116 @@ def get_table_fqn_from_query_name(
     return database_query, schema_query, table
 
 
+def handle_udf_column_lineage(
+    column_lineage_original: dict,
+    column_lineage_generated: List[Tuple[Column, Column]],
+):
+    """
+    Handle UDF column lineage
+    """
+    try:
+        result = {}
```

Comment:
you can make result a `from collections import defaultdict; result = defaultdict(dict)` so you don't need the line on ln 188.